testdata/catalogs/test-catalog/catalog.yaml

```diff
@@ -0,0 +1,48 @@
+---
+schema: olm.package
+name: prometheus
+defaultChannel: beta
+---
+schema: olm.channel
+name: beta
+package: prometheus
+entries:
+  - name: prometheus-operator.0.47.0
+---
+schema: olm.bundle
+name: prometheus-operator.0.47.0
+package: prometheus
+image: localhost/testdata/bundles/registry-v1/prometheus-operator:v0.47.0
+relatedImages:
+  - name: ""
+    image: ""
+properties:
+  - type: olm.package
+    value:
+      packageName: prometheus
+      version: 0.47.0
+---
+schema: olm.package
+name: plain
+defaultChannel: beta
+---
+schema: olm.channel
+name: beta
+package: plain
+entries:
+  - name: plain.0.1.0
+---
+schema: olm.bundle
+name: plain.0.1.0
+package: plain
+image: localhost/testdata/bundles/plain-v0/plain:v0.1.0
+relatedImages:
+  - name: ""
+    image: ""
```

Comment:
We could remove this if we merge https://github.com/operator-framework/operator-controller/pull/259 first.